doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -336,6 +336,7 @@ The `mz_kafka_sources` table contains a row for each Kafka source in the system.
 |------------------------|----------------|-----------------------------------------------------------------------------------------------------------|
 | `id`                   | [`text`]       | The ID of the Kafka source. Corresponds to [`mz_catalog.mz_sources.id`](../mz_catalog#mz_sources).        |
 | `group_id_prefix`      | [`text`]       | The value of the `GROUP ID PREFIX` connection option.                                                     |
+| `topic          `      | [`text`]       | The topic this Kafka source is reading from.                                                              |
```

Comment:
```suggestion
| `topic          `      | [`text`]       | The name of the Kafka topic the source is reading from.                                                              |
```

Reply:
🫠 nit. For consistency with the description in `mz_kafka_sinks`.

Reply:
rip